src/modules/job-info/job_state.c

```diff
@@ -1208,6 +1208,7 @@ static void update_jobs (struct info_ctx *ctx, json_t *transitions)
                 job_destroy (job);
                 return;
             }
+            flux_log_error (jsctx->h, "%s: job id %ju inserted", __FUNCTION__, (uintmax_t)job->id);
```

Comment:
Should this be a just a normal log (rather than an error)?

Reply:
oh crap, that's my error debugging!